sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/util/Utility.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.implementation.util;
+
+import com.azure.core.experimental.serializer.JsonSerializer;
+import com.azure.core.serializer.json.jackson.JacksonJsonSerializerBuilder;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.search.documents.implementation.serializer.SerializationUtil;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public final class Utility {
+    /**
+     * Helper class to initialize the SerializerAdapter.
+     * @return The SerializeAdapter instance.
+     */
+    public static SerializerAdapter initializeSerializerAdapter() {
+        JacksonAdapter adapter = (JacksonAdapter) JacksonAdapter.createDefaultSerializerAdapter();
+
+        ObjectMapper mapper = adapter.serializer();
```

Comment:
`.copy()` the serializer otherwise this will mutate the Azure Core serializer which we won't want. Another option would be creating a new instance of `JacksonAdapter` rather than use `createDefaultSerializerAdapter` (which creates or returns the global instance depending on whether it has already been instantiated).

Reply:
Is there an API for .copy()? 

The API here is to use across the rest client(core serializer). I would like to share cross the clients to save the initialization time as they always use the same settings.
For the customer serializer layer, we have a new instance to make sure core serializer doesn't mix the use with it.

Reply:
There is an issue with changing this for all clients as this could affect how AppConfiguration, KeyVault, Storage, etc serialize and deserialize when used along side Search which mutates the global serializer. Maybe this method should be turned into a lazy loaded which initializes a constant `ObjectMapper` used for Search instead of changing the global `ObjectMapper`.